yalu102/jailbreak.m

```diff
@@ -474,30 +474,30 @@ void exploit(void* btn, mach_port_t pt, uint64_t kernbase, uint64_t allprocs)
     
     
 #define RemapPage(address) \
-    pagestuff_64((address) & (~PMK), ^(vm_address_t tte_addr, int addr) {\
-        uint64_t tte = ReadAnywhere64(tte_addr);\
-        if (!(TTE_GET(tte, TTE_IS_TABLE_MASK))) {\
-            NSLog(@"breakup!");\
-            uint64_t fakep = physalloc(PSZ);\
-            uint64_t realp = TTE_GET(tte, TTE_PHYS_VALUE_MASK);\
-            TTE_SETB(tte, TTE_IS_TABLE_MASK);\
-            for (int i = 0; i < PSZ/8; i++) {\
-                TTE_SET(tte, TTE_PHYS_VALUE_MASK, realp + i * PSZ);\
-                WriteAnywhere64(fakep+i*8, tte);\
-            }\
-            TTE_SET(tte, TTE_PHYS_VALUE_MASK, findphys_real(fakep));\
-            WriteAnywhere64(tte_addr, tte);\
-        }\
-        uint64_t newt = physalloc(PSZ);\
-        copyin(bbuf, TTE_GET(tte, TTE_PHYS_VALUE_MASK) - gPhysBase + gVirtBase, PSZ);\
-        copyout(newt, bbuf, PSZ);\
-        TTE_SET(tte, TTE_PHYS_VALUE_MASK, findphys_real(newt));\
-        TTE_SET(tte, TTE_BLOCK_ATTR_UXN_MASK, 0);\
-        TTE_SET(tte, TTE_BLOCK_ATTR_PXN_MASK, 0);\
-        WriteAnywhere64(tte_addr, tte);\
-        NSLog(@"level %llx - %llx", tte_addr,              TTE_GET(tte, TTE_PHYS_VALUE_MASK));\
-    }, level1_table, 2);
-
+pagestuff_64((address) & (~PMK), ^(vm_address_t tte_addr, int addr) {\
```

Comment:
Don't mess with the indentation in the code, as it's irrelevant to the change you are describing.